docs/3.tutorials/fts/simple-fts.md

```diff
@@ -4,7 +4,7 @@ title: Minting Fungible Tokens
 ---
 
 In this tutorial, you'll learn how to easily create your own Fungible Tokens (FT) without doing any software development by using a readily-available smart contract.
-This article will show you how to use an existing [FT smart contract](#fungible-token-contract), and you'll learn [how to mint](#minting-your-fts) fungible tokens and [view them](#checking-your-wallet) in your Wallet.
+This article will show you how to use an existing [FT smart contract](https://docs.near.org/tutorials/fts/skeleton), and you'll learn [how to mint](#minting-your-fts) fungible tokens and [view them](#checking-your-wallet) in your Wallet.
```

Comment:
```suggestion
This article will show you how to use an existing [FT smart contract](0-predeployed.md), and you'll learn [how to mint](#minting-your-fts) fungible tokens and [view them](#checking-your-wallet) in your Wallet.
```

Reply:
Agreed i was thinking that the tutorial is at the step where we can deploy our own contract instead of using the predeplolyed one